network-area-diagram-viewer/src/main/resources/svg.js

```diff
@@ -0,0 +1,448 @@
+
+// TODO(Luma) consider three winding transformers
+// TODO(Luma) consider nodes with multiple buses
+// TODO(Luma) separate components of edges, having one that can be "scaled" (length line from exit of node to middle point)
```

Comment:
That seems very promising! because of the two previous TODOs, we cannot achieve everything with that, but still, it makes it a lot simpler.

Reply:
I've added a scaling that is _correct_ but does not take into account _stretchable_ parts, just to verify the combination of transformations and the basic calculations. Again, maybe we could merge the current version and deal with this in separate PR